phpunit-deprecation-baseline.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<files version="1">
+ <file path="vendor/symfony/deprecation-contracts/function.php">
+  <line number="25" hash="c6af5d66288d0667e424978000f29571e4954b81">
+   <issue><![CDATA[Since symfony/framework-bundle 6.4: Not setting the "framework.php_errors.log" config option is deprecated. It will default to "true" in 7.0.]]></issue>
```

Comment:
I don't know why, the CI fails on 6.4 when I add this option to the framework.... :shrug: 

But I know the lib is working with the option enabled, hence the exception here